src/components/BTextArea/BTextArea.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div :class="classes">
+    <label v-if="!hideLabel" :for="id">
+      <strong> 
+        {{ label }}<span v-if="required" aria-label="Required">*</span>
+      </strong>   
+    </label>
+    <textarea
+      v-model="inputValue"
+      v-bind="{id, name, cols, rows, placeholder, required, disabled}"
+      :placeholder="placeholder ? placeholder : null"
+      :aria-label="hideLabel ? label : null"
+    />
+    <div class="b-input-group--error">
```

Comment:
This may be a whole other level of detail that we really don't need to care about - but does anyone have opinions about using single quotes for strings since double quotes are used for directives?

Reply:
It's weird to me to not use double quotes in HTML.